src/templates/ObservabilityPackDetails.js

```diff
@@ -230,6 +250,28 @@ const ObservabilityPackDetails = ({ data, location }) => {
               }
             `}
           >
+            <PageTools.Section
+              css={css`
+                background-color: var(--divider-color);
+              `}
+            >
+              <div>
+                <Button
```

Comment:
It could be worth adding some event tracking for this button click, just an idea!

Reply:
added